mso/resource_mso_schema_template_anp_epg.go

```diff
@@ -112,6 +112,67 @@ func resourceMSOSchemaTemplateAnpEpg() *schema.Resource {
 				Optional: true,
 				Computed: true,
 			},
+			"epg_type": &schema.Schema{
+				Type:     schema.TypeString,
+				Optional: true,
+				ValidateFunc: validation.StringInSlice([]string{
+					"application",
+					"service",
+				}, false),
+				Default: "application",
+			},
+			"cloud_service_epg_config": &schema.Schema{
```

Comment:
Why having those as a subresource and not just have the 4 arguments directly in the resource itself?

Reply:
Mimicked the API payload.

Reply:
Let's do like we did for private_link_label and put them at the epg level and not a subresource.

Reply:
ok